Makefile

```diff
@@ -14,19 +13,15 @@ venv/bin/activate: requirements.txt
 	.venvbag/bin/pip install -Ur requirements.txt
 	touch .venvbag/bin/activate
 
-% : tests/test_%.py
-	${RUNTEST} test_$@
-
-deploy-dev: venv
-	@echo "running tests..."
+tests: venv
+	@echo "in tests"
 	.venvbag/bin/python ${RUNTEST} ${ALLMODULES}
 	.venvbag/bin/pip install awscli --upgrade
```

Comment:
While I understand wanting to avoid duplication, it is still confusing to have a target called tests that installs the AWS client.

Can you create another target that has a dependency on tests? Something like:

```
tests: venv
     @echo "Running tests"
      ....

aws-cli: tests
   .venvbag/bin/pip install awscli --upgrade

deploy-dev: aws-cli